client/screens/Navigation.js

```diff
@@ -27,6 +27,7 @@ import Drawer from "./Drawer";
 import { Link, useRouter, usePathname } from "expo-router";
 import { hexToRGBA } from "../utils/colorFunctions";
 import UseTheme from "../hooks/useTheme";
+import { useRoute } from "@react-navigation/native";
```

Comment:
We should probably be using expo router here

Reply:
I think?

Reply:
yes

Reply:
it's only import neither use it anywhere.